src/tests/update_tasks.test.js

```diff
@@ -0,0 +1,36 @@
+/**      A function for editing the task description.       */
+import { addTask, removeTask } from "./test_files/add_remove";
+import { displayTasks, tasks } from "./test_files/index";
+
+jest.mock("./test_files/add_remove");
+
+displayTasks();
+
+describe("Test task manager ->", () => {
+  test("add 1 li element to the ul", () => {
+    const input = global.document.getElementById("description");
+    input.value = "Do stuff";
+    const task = { id: 4, index: 4, description: "Do stuff", completed: true };
```

Comment:
- [OPTIONAL]: 'task' is declared but its value is never read. I delete this line and the project works yet.

Reply:
Thanks for pointing out Victor. I take note of that behavior. Thanks for your time, I really appreciate it. Have a nice day.